ghtt/config.py

```diff
@@ -140,7 +143,37 @@ def get_repos(students: List[Person], mentors: Optional[List[Person]] = None) ->
         repo.organization = organization
         repo.mentors = [m for m in mentors if repo.group in m.groups]
         repos[repo.name] = repo
-    return repos
+
+    # Check if all repo's have expected number of students/mentors
```

Comment:
I'm not convinced this should be in config.py because then it's used for every action. I think this is only relevant for the `create_*` actions. The other actions like `grant` and `pull` should just use all available repositories. I want `pull`, `grant` and `remove-grant` to run as quickly as possible and without user interaction because these are used at the start and end of labs/exams and in case something goes horribly wrong and we need to remove students' access immediately.

Reply:
Maybe we can add a flag "size_ok", or something to the repo object here? That we then check in the `create_*` functions and ask the user about if size is not ok. The other functions then just ignore that flag.